workflow-dev/tpl/generate_params.toml

```diff
@@ -25,8 +25,10 @@ pullPolicy = "Always"
 dockerTag = "canary"
 
 [s3]
-accesskey = "YOUR KEY HERE"
-secretkey = "YOUR SECRET HERE"
+#Your aws access key.Leave it empty if you are on aws and want to use instance profile
```

Comment:
also needs a space between the period and the "Leave", as well as one after the comment hash mark for punctuation.